python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_name_server_groups.schema.yml

```
@@ -0,0 +1,44 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ip_name_server_groups:
+    type: list
+    primary_key: name
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+        name_servers:
+          type: list
+          primary_key: ip_address
+          allow_duplicate_primary_key: true
+          items:
+            type: dict
+            keys:
+              ip_address:
+                type: str
+                description: IPv4 or IPv6 address for DNS server.
+              vrf:
+                description: VRF Name.
+                type: str
+                default: default
+                convert_types:
+                  - int
+              priority:
+                description: Priority value (lower is first).
+                type: int
+                convert_types:
+                  - str
+                min: 0
+                max: 4
```

```suggestion
                max: 15
```
In latest version of EOS it has max value as 15

---

Fixed.